### DISCUSSION TITLE
Issue with role

### DISCUSSION THREAD

**Discussion by OMacload** [almenscorner/IntuneCD#87] [Q&A] [answered]
Hi, I did the installation but i have a problem with role..

The message :

Authentication Failed
User does not have a role assigned.

Thanks for your help :)

**Comment by almenscorner** [comment]
Hi.

I'm guessing you get this message when trying to sign into [IntuneCD Monitor](https://github.com/almenscorner/intunecd-monitor)?

If so, have you applied the role as described [here](https://github.com/almenscorner/IntuneCD-monitor#deploy-to-azure)?

**Comment by OMacload** [comment]
Hi, yes it s for IntuneCD Monitor

I removed my account and added again.

Now my account have the good role : 

Roles:   intunecd-admin

Thank you :)

But I want to check settings and i have this message :

No permission
Admin role are required to access this page.

Regards

**Comment by almenscorner** [comment, marked as answer]
To access settings within IntuneCD Monitor, your account must be assigned the admin role and not just user. By default the name must be "intunecd_admin" and be assigned on your account on the Azure AD app like this:

![Screenshot 2023-01-25 at 14 38 47](https://user-images.githubusercontent.com/78877636/214578474-2c2710be-d23a-47ab-8612-fd3189edb4dc.png)

**Comment by OMacload** [comment]
![image](https://user-images.githubusercontent.com/123559377/214581087-f238069f-2cb4-49b6-8c68-c970d4d319de.png)

Strange I can't change my role on the apps.

I will search..  :)

**Comment by OMacload** [comment]
It s good I did an error with the intunecd_admin account name like intunecd-admin :) 

Many thanks :)
